### PR TITLE
Update 0C0A.lang

### PR DESCRIPTION
**Pull request by JeffersonRussell**
Fixed errors
Fixed long texts
Improvements

### CODE REVIEW

**Comment by micwoj92** [comment]
@97carmine Do you approve?

File: Langs/0C0A.lang [hunk 1]
```diff
@@ -1,59 +1,64 @@
...
+Large=18
+Medium=10
+Small=8.5
 [Strings]
 Donate=Donar
-Check for Updates=Comprobar actualizaciones
-Your Windows 11 Compatibility Results are Below=Los resultados de compatibilidad de Windows 11 están abajo 
-Now Reach WhyNotWin11 via https://www.whynotwin11.org/=Ahora llega a WhyNotWin11 a través de https://www.whynotwin11.org/
-Results Based on Currently Known Requirements!=¡Resultados basados en requisitos actualmente conocidos!
+Check for Updates=Actualizar
```
Comment (97carmine) [inline Langs/0C0A.lang:12]: My translation is more accurate

File: Langs/0C0A.lang [hunk 2]
```diff
@@ -1,59 +1,64 @@
...
 Translation by=Traducción por
 Architecture (CPU + OS)=Arquitectura (CPU + SO)
-Boot Method=Método de arranque
-CPU Compatibility=Compatibilidad con CPU
-CPU Core Count=Conteo de núcleos del CPU
-CPU Frequency=Frecuencia del CPU
+Boot Method=Método de Arranque
+CPU Compatibility=Compatibilidad de CPU
+CPU Core Count=Núcleos de CPU
+CPU Frequency=Frecuencia de CPU
```
Comment (97carmine) [inline Langs/0C0A.lang:21]: The problem is that it does not respect grammar, it enters capital letters that should be lowercase letters

File: Langs/0C0A.lang [hunk 3]
```diff
@@ -1,59 +1,64 @@
...
 DirectX + WDDM2=DirectX + WDDM2
-Disk Partition Type=Tipo de partición de disco
-RAM Installed=RAM instalada
-Secure Boot=Arranque seguro
-Storage Available=Almacenamiento disponible
+Disk Partition Type=Tipo de Partición de Disco
+RAM Installed=RAM Instalada
+Secure Boot=Arranque Seguro
+Storage Available=Espacio Disponible
```
Comment (97carmine) [inline Langs/0C0A.lang:26]: Same problem as above

File: Langs/0C0A.lang [hunk 4]
```diff
@@ -1,59 +1,64 @@
...
-Unable to Check List=No se puede revisar la lista
-Error Accessing List=Error al acceder a la lista
-Not Currently Listed as Compatible=Actualmente no se muestra como compatible
-Listed as Compatible=Listado como compatible
+32 Bit OS=SO de 32 Bits
+Unable to Determine=Incapaz de Determinar
+Unable to Check List=Incapaz de Comprobar Listado
+Error Accessing List=Error al Acceder al Listado
+Not Currently Listed as Compatible=Actualmente no Listado Como Compatible
+Listed as Compatible=Listado Como Compatible
```
Comment (97carmine) [inline Langs/0C0A.lang:37]: Same problem

File: Langs/0C0A.lang [hunk 5]
```diff
@@ -1,59 +1,64 @@
...
 Cores=Núcleos
 Threads=Hilos
-GPT Detected=GPT detectado
-GPT Not Detected=GPT no detectado
-Enabled=Habilitado
+GPT Detected=GPT Detectado
+GPT Not Detected=GPT No Detectado
+Enabled=Activado
```
Comment (97carmine) [inline Langs/0C0A.lang:42]: Same problem

File: Langs/0C0A.lang [hunk 6]
```diff
@@ -1,59 +1,64 @@
...
 Supported=Soportado
-Not Supported=No soportado
-Disabled / Not Detected=Desabilitado / No detectado
-Drive(s) Meet Requirements=Disco(s) cumple con los requisitos
-TPM Missing / Disabled=TPM Ausente / Desactivado
+Not Supported=No Soportado
+Disabled / Not Detected=Desactivado / No Detectado
+Drive(s) Meet Requirements=Unidad(es) Compatible(s)
```
Comment (97carmine) [inline Langs/0C0A.lang:46]: Same problem

File: Langs/0C0A.lang [hunk 7]
```diff
@@ -1,59 +1,64 @@
...
-No DirectX 12, but WDDM2=No DirectX 12, pero WDDM2
-DirectX 12, but no WDDM2=DirectX 12, pero no WDDM2
-No DirectX 12 or WDDM2=No DirectX 12 o WDDM2
-Test Build?=¿Probar lanzamiento?
-You're running a newer build than publicly Available!=¡Estás ejecutando un lanzamiento mayor que el públicamente disponible!
+No DirectX 12, but WDDM2=Sin DirectX 12, pero con WDDM2
+DirectX 12, but no WDDM2=DirectX 12, pero sin WDDM2
+No DirectX 12 or WDDM2=Sin DirectX 12 o WDDM2
+Test Build?=¿Probar Versión?
+You're running a newer build than publicly Available!=¡Estás Ejecutando una Versión más Nueva que la Disponible Públicamente!
```
Comment (97carmine) [inline Langs/0C0A.lang:55]: Same problem

File: Langs/0C0A.lang [hunk 8]
```diff
@@ -1,59 +1,64 @@
...
-Update Available=Actualización no disponible
-An Update is Available, would you like to download it?=Una actualización está disponible, ¿le gustaría descargarlo?
+You're running the latest build!=¡Estás Ejecutando la Última Versión!
+Unable to Check for Updates=Incapaz de Comprobar las Actualizaciones
+Unable to load release data.=Incapaz de Cargar Datos de la Versión
+Invalid Data Received!=¡Se han Recibido Datos no Válidos!
+Invalid Release Tags Received!=¡Se han Recibido Etiquetas de Versión no Válidas!
+Invalid Release Types Received!=¡Se han Recibido Tipos de Versión no Válidas!
+Update Available=Actualización Disponible
+An Update is Available, would you like to download it?=Hay una Actualización Disponible, ¿Le Gustaría Descargarla?
```
Comment (97carmine) [inline Langs/0C0A.lang:64]: Same problem

**Review by 97carmine — COMMENTED** [review]
Except for the first revision, the others indicate the same problem, grammar is not respected, it enters capital letters that should be lowercase letters.

File: Langs/0C0A.lang (hunk 1 above)
Comment (JeffersonRussell) [reply Langs/0C0A.lang:12]: Line 8: The sentence is too long.
Line 9: The beginning of the sentence does not make sense.
Line 10: I capitalized the same words as in the original sentence.

File: Langs/0C0A.lang (hunk 2 above)
Comment (JeffersonRussell) [reply Langs/0C0A.lang:21]: I capitalized the same words as in the original sentence in all translations.

File: Langs/0C0A.lang (hunk 1 above)
Comment (micwoj92) [reply Langs/0C0A.lang:12]: Line 10: don't have to translate 1:1, follow your language's rules.

Comment (97carmine) [reply Langs/0C0A.lang:12]: If a translation 1:1 is not sought, then the translation is fine

Comment (micwoj92) [reply Langs/0C0A.lang:12]: Thank you, I will merge this then. Feel free to submit later improvements

**Pull request merged**